rootfs/usr/share/www/observer.css

```
@@ -28,8 +28,14 @@ body {
 .disconnected {
     color: red;
 }
+.reason {
```

Override the link color explicitly.

```suggestion
.reason a {
```

---

That excludes the dot